src/components/date-time-input/date-time-input.ts

```diff
@@ -0,0 +1,652 @@
+import { html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { live } from 'lit/directives/live.js';
+import {
+  DatePartDeltas,
+  DatePartInfo,
+  DateParts,
+  DatePart,
+  DateTimeUtil,
+} from './date-util';
+import { blazorTwoWayBind, watch } from '../common/decorators';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { IgcMaskInputBaseComponent } from '../mask-input/mask-input-base';
+import { partNameMap } from '../common/util';
+import { IgcInputEventMap } from '../input/input-base';
+import { EventEmitterMixin } from '../common/mixins/event-emitter';
+import { AbstractConstructor } from '../common/mixins/constructor';
```

Comment:
Please fix the import paths for ESM resolution.

For concrete imports add the relevant path suffixes `.js`.
For types, you can omit the suffix if you follow the `import type {T} from 'path'`